### PR TITLE
Packages

### PR DESCRIPTION
**Pull request by Easiv**
Added required dependencies

### CODE REVIEW

**Review by Rxbsxn — CHANGES_REQUESTED** [review]
I think you have missed some of the packages, please take a look at the task requirements

**Comment by Easiv** [comment]
@Rxbsxn - applied feedback

File: Gemfile [hunk 1]
```diff
@@ -3,10 +3,12 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.5.1'
 
+gem 'rspec'
```
Comment (Rxbsxn) [inline Gemfile]: Do we need rspec running on production or rather we are going to use it in testing/development environment? So I would move it to testing/dev section. Also please change `rspec` -> `rspec-rails`

**Review by Rxbsxn — CHANGES_REQUESTED** [review]
feedback once again

**Comment by Rxbsxn** [comment]
And please resolve conflicts

**Comment by Easiv** [comment]
@Rxbsxn - done again

File: Gemfile [hunk 2]
```diff
@@ -31,10 +31,16 @@ gem 'bootsnap', '>= 1.1.0', require: false
...
+  gem 'database_cleaner'
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'shoulda-matchers', '~> 3.1'
+  gem 'factory_bot_rails'
+  gem 'ffaker'
+  
 end
 
 group :development do
+  gem 'rspec-rails'
```
Comment (Rxbsxn) [inline Gemfile]: it should be in `:development, :test` group

**Review by Rxbsxn — CHANGES_REQUESTED** [review]
one more thing to do

File: Gemfile (hunk 2 above)
Comment (PoslinskiNet) [reply Gemfile]: Or even in `:test` group only.

**Comment by Easiv** [comment]
Should be done now

File: Gemfile.lock [hunk 3]
```diff
@@ -139,11 +166,19 @@ PLATFORMS
 DEPENDENCIES
   bootsnap (>= 1.1.0)
   byebug
+<<<<<<< HEAD
+  database_cleaner
+  factory_bot_rails
```
Comment (Rxbsxn) [inline Gemfile.lock]: remove gemfile.lock -> bundle i

File: Gemfile [hunk 4]
```diff
@@ -31,7 +31,12 @@ gem 'bootsnap', '>= 1.1.0', require: false
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
+  gem 'database_cleaner'
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'shoulda-matchers', '~> 3.1'
```
Comment (Rxbsxn) [inline Gemfile]: database_cleaner, shoulda-matcher, ffaker, factory_bot_rails -> group :test

**Review by Rxbsxn — CHANGES_REQUESTED** [review]

**Comment by Easiv** [comment]
@Rx🅱️sxn -> Checc this out

**Comment by Rxbsxn** [comment]
💯

**Pull request merged**